test_agent/python/testagent.py

```diff
@@ -65,31 +60,36 @@
 logger = logging.getLogger("File:Line# Debugger")
 logger.setLevel(logging.DEBUG)
 
+RESPONSE_URI = UUri(ue_id=1, ue_version_major=1, resource_id=0)
+
+sdkname = "python"
+transport_name = "socket"
```

Comment:
Not used anywhere

Reply:
will be used in the future, once more transports are added to python, so would like to keep it